cshub-server/src/db/entities/emaildomain.ts

```diff
@@ -0,0 +1,25 @@
+import { Exclude, Expose } from "class-transformer";
+import {Column, Entity, OneToMany, PrimaryGeneratedColumn} from "typeorm";
+import { IEmailDomain } from "../../../../cshub-shared/src/entities/emaildomains";
+import {User} from "./user";
+
+@Exclude()
+@Entity({
+    name: "emaildomains"
+})
+export class EmailDomain implements IEmailDomain {
+    @Expose()
+    @PrimaryGeneratedColumn()
+    id!: number;
+
+    @Expose()
+    @Column({
+        type: "varchar",
+        length: 64,
+        unique: true
+    })
+    domain!: string;
+
+    @OneToMany(type => User, user => user.domain)
+    users?: User[];
```

Comment:
Shouldn't a user have a domain instead of a domain having users?

Reply:
good point

Reply:
that's why it wasn't working

Reply:
I'll take a look at this later, but this should be fine. You should make these joins both ways in typeorm